sim/validator.py

```diff
@@ -141,8 +174,8 @@ def main():
     # Parse
     parser = argparse.ArgumentParser(description="Performance test btb-seq code")
     parser.add_argument("btb_seq", help="path to btb-seq code")
-    parser.add_argument("results", help="path to performance test results")
-    parser.add_argument("--branch", help="name of btb-seq branch to use", default=None)
+    parser.add_argument("output_path", help="path to performance test results")
+    parser.add_argument("--branch", help="name of btb-seq branch to use", default='master')
```

Comment:
I prefer to have this as `None`. Sometimes you want to test something without having to check it into source.